apps/cookbook/src/app/showcase/modal-showcase/modal-showcase.component.ts

```diff
@@ -67,6 +67,11 @@ export class ModalShowcaseComponent implements AfterViewInit {
       description: 'Emitted when the modal is ready and the enter animation has finished',
       inputValues: ['Promise<void>'],
     },
+    {
+      name: 'willDismiss',
```

Comment:
I know this is the Ionic jargon, but maybe align with existing Kirby wording?
```suggestion
      name: 'willClose',
```